From a305c9fcded090a9cd4d63b00ea826787422cb00 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Mon, 13 May 2024 14:51:46 -0700
Subject: [PATCH] Use the suds community fork

---
 PKG-INFO | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/PKG-INFO b/PKG-INFO
index dc087ad98609..b91921df10f1 100644
--- a/PKG-INFO
+++ b/PKG-INFO
@@ -8,6 +8,6 @@ License-File: COPYING
 Requires-Dist: pyserial
 Requires-Dist: requests
 Requires-Dist: yattag
-Requires-Dist: suds
+Requires-Dist: suds_community
 Provides-Extra: wx
 Requires-Dist: wxPython; extra == "wx"
-- 
2.45.0

